src/xlsx/mod.rs

```diff
@@ -2199,6 +2435,474 @@ pub(crate) fn path_to_zip_path<RS: Read + Seek>(zip: &ZipArchive<RS>, path: &str
     path.to_string()
 }
 
+pub mod pivot_cache {
+    use super::XlReader;
+    use crate::{CellErrorType, Data, XlsxError};
+    use quick_xml::events::attributes::Attribute;
+    use quick_xml::events::BytesStart;
+    use quick_xml::events::Event;
+    use quick_xml::name::QName;
+    use quick_xml::Decoder;
+    use std::collections::HashMap;
+    use std::io::{Read, Seek};
+
+    pub type Tag = Box<[u8]>;
+    pub type Value = Option<Box<[u8]>>;
+
+    // Get the target location of the pivot table's pivot cache definitions.
+    pub fn find_pivot_cache_definitions_from_pivot<RS>(
```

Comment:
Public functions should have full documentation (some of the functions elsewhere in this commit do and some don't.

Reply:
So should it be `pub(crate)` then?